packages/generator/src/generator.ts

```diff
@@ -99,7 +105,16 @@ export abstract class Generator<T extends GeneratorOptions = GeneratorOptions> e
         const templateValues = await this.getTemplateValues()
 
         this.fs.copy(this.sourcePath(filePath), this.destinationPath(pathSuffix), {
-          process: (input) => this.process(input, pathSuffix, templateValues),
+          process: (input) => {
+            let formattedData = this.process(input, pathSuffix, templateValues)
+
+            if (prettierExtensions.test(pathSuffix) && typeof formattedData === 'string') {
+              log.progress(`Formatting file ${pathSuffix}`)
```

Comment:
Can we remove this log line for now? I think prettier can be a silent step since most users would expect that we're generating formatted files